causal_testing/testing/causal_test_outcome.py

```diff
@@ -74,6 +85,7 @@ def apply(self, res: CausalTestResult) -> bool:
             return res.test_value.value > 0
         if res.test_value.type == "risk_ratio":
             return res.test_value.value > 1
+        # Dead code but necessary for pylint
```

Comment:
Can we ensure that in all scenarios at least one of the above `if` statements will definitely run?

If a user misconfigures something then this error may still be helpful

Reply:
Yes, because of the call to `super` in the first `if` statement. If the user misconfigures `res.test_value.value` then this will throw a `ValueError` from there. I'm pretty sure it's impossible to reach line 89 unless we change the `super` method later on.